jdaviz/configs/imviz/plugins/aper_phot_simple/aper_phot_simple.py

```diff
@@ -387,15 +380,19 @@ def vue_do_aper_phot(self, *args, **kwargs):
                             f"Radial profile fitting: {msg}", color='warning', sender=self))
                     y_fit = fit_model(x_data)
                     self.app.fitted_models[self._fitted_model_name] = fit_model
-                    fit_line.x, fit_line.y = x_data, y_fit
+                    self.plot._update_data('fit', x=x_data, y=y_fit)
+                    self.plot.update_style('fit', color='magenta',
+                                           markers_visible=False, line_visible=True)
                 else:
-                    self.plot.clear_marks('fit_line')
+                    self.plot.update_style('fit', visible=False)
 
         except Exception as e:  # pragma: no cover
-            self.plot.clear_all_marks()
+            self.plot.update_style('profile', visible=False)
+            self.plot.update_style('fit', visible=False)
             msg = f"Aperture photometry failed: {repr(e)}"
             self.hub.broadcast(SnackbarMessage(msg, color='error', sender=self))
             self.result_failed_msg = msg
+            raise

```

Comment:
What is the point of emitting snackbar message if you are going to throw the exception here anyway?

Reply:
Woops, this was just for debugging and I forgot to remove it. Thanks for noticing!